style.css

```diff
@@ -0,0 +1,213 @@
+:root{
+    --startBarColor:rgb(30 28 28);
+}
+
+body{
+    background-image: url("/html-project-rickcedwhat/images/windows desktop.jpg");
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    min-height: 100vh;
+    font-family: system-ui;
+}
+
+main{
+    display: flex;
+    flex-direction: column;
+    height:100vh;
+}
+
+#desktop{
+    overflow-y: clip;
+}
+
+#desktop #icons{
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    height: calc(100vh - 44px);
+}
+
+.desktop-icon{
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+
+.desktop-icon img{
+    height:75%;
+    cursor:pointer;
+    
+}
+
+.toolbar-icon img{
+    height:100%;
+    cursor:pointer;
+}
+
+#start-bar{
+    display:flex;
+    background-color: var(--startBarColor)
+}
+
+.desktop-icon{
+    height:75px;
+    width:75px;
+    padding:5px;
+}
+
+ul{
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    align-items:center;
+}
+
+.icon{
+    display:flex;
+    justify-content: center;overflow-y: clip;
+    align-items:center;
+}
+
+#searchbar input{
+    width:281px;
+    height:41px;
+    font-size:20px;
+    margin-right:5px;
+}
+
+::placeholder{
+    color:#575151;
+}
+
+input:focus{
+    outline:none;
+}
+
+#window-objects{
+    position:absolute;
+    top:0;
+    bottom:0;
+    left:0;
+    right:0;
+    pointer-events: none;
+}
+
+.window-object{
+    position:absolute;
+    display:none;
+    background:white;
+    height:attr(height);
+    width:attr(width);
+    pointer-events: all;
+}
+
+.open{
+    display:flex;
+    flex-direction: column;
+}
+
+.maximized{
+    top:0;
+    left:0;
+    right:0;
+    bottom:43px;
+}
+
+.border-bar{
+    display: flex;
+    justify-content: space-between;
+    width:100%;
+    align-items: center;
+    color:white;
+}
+
+.border-bar ul{
+    display: flex;
+}
+
+.border-bar li{
+    margin: 5px;
+    cursor: pointer;
+    width: 40px;
+    height: 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition-duration: 0.4s;
+}
+
+ 
+
+.border-bar li:hover{
+    background-color: rgb(37, 37, 37);
+    color: white;
+}
+
+.toolbar-icon{
+    display:none;
+    cursor: pointer;
+    height:45px;
+    margin-left:5px;
+}
+
+.show, .pinned{
+    display:block;
+}
+
+.show{
+    border-bottom:2px solid lightblue;
+}
+
+.active{
+    background:#454444;
+}
+
+.title{
+    padding:0 10px;
+}
+
+.body{
+    margin:10px;
+    overflow-y:scroll;
+}
+
+.container{
+    display: flex;
+}
+
+.column{
+    flex-direction: column;
+}
+
+.space-evenly{
```

Comment:
these classes are an interesting way to almost reflect style-named CSS classes such as those used in Bootstrap!